pkg/operator/operator.go

```diff
@@ -294,7 +321,15 @@ func (optr *Operator) Run(workers int, stopCh <-chan struct{}) {
 		optr.mcoSAListerSynced,
 		optr.mcoSecretListerSynced,
 		optr.ocSecretListerSynced,
-		optr.mcoCOListerSynced) {
+		optr.mcoCOListerSynced}
+	fg, err := optr.fgAccessor.CurrentFeatureGates()
+	if err != nil {
+		klog.Errorf("No fg enabled %w", err)
```

Comment:
When does this error again? (Also nit: indentation)

Reply:
when the FG accessor is not ready, happens during some weird race if the rest of the cluster is having installation issues.